.github/workflows/ci-cd-pipeline.yml

```diff
@@ -3,16 +3,20 @@ name: CI/CD pipeline
 env:
   DOCKER_REGISTRY: nethermind.jfrog.io
 
-  REPO_DEV: nubia-docker-local-dev
-  REPO_STAGING: nubia-docker-local-staging
-  REPO_PROD: nubia-docker-local-prod
+  REPO_DEV: nubia-oci-local-dev
+  REPO_STAGING: nubia-oci-local-staging
+  REPO_PROD: nubia-oci-local-prod
 
 
 on:
   push:
     branches: [main]
     tags: ["v*"]
   workflow_dispatch:
+  pull_request:
+    branches: [main]
```

Comment:
do we want to promote/deploy to dev for each PR to main?

Reply:
nope, this is only for PRs that touch the CI file, it's a way to ensure the changes work as expected.  I can also remove this part and just trust that it works, or just manually running via the workflow_dispatch